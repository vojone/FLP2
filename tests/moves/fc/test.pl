:- consult('../../../cube').

:- read_cube(C, _), move_f_ccw(C, _, _, RC), write_cube(RC).
