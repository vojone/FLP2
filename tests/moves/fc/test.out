123
456
122
411 311 222 339
411 222 333 448
411 122 333 447
344
666
666
